### PR TITLE
Added ClientServer#isTransferring

### PR DESCRIPTION
**Pull request by lokerhp**
I added ClientServer#isTransferring in this PR, isInstalling and isSuspended already existed so I added isTransferring

### CODE REVIEW

File: src/main/java/com/mattmalec/pterodactyl4j/client/entities/impl/ClientServerImpl.java [hunk 1]
```diff
@@ -117,6 +117,11 @@ public boolean isInstalling() {
 		return json.getBoolean("is_installing");
 	}
 
+	@Override
+	public boolean isTransferring() {
+		return json.getBoolean("is_transferring");
```
Comment (mattmalec) [inline src/main/java/com/mattmalec/pterodactyl4j/client/entities/impl/ClientServerImpl.java:122]: Can you show where this is returned in the client API? Just want to see the raw JSON for this object.

**Review by mattmalec — CHANGES_REQUESTED** [review]
See the commit comment for details.

**Comment by lokerhp** [comment]
I added a commit which also fixes a small Javadoc mistake I noticed.

I've also sended the raw JSON data in your Discord DM @mattmalec

**Review by mattmalec — APPROVED** [review]
Thanks! This will work.

**Comment by mattmalec** [comment]
Thanks for fixing that documentation too, good catch.

**Pull request merged**